config/core/configmaps/gc.yaml

```
@@ -60,27 +60,58 @@ data:
     # V2 Garbage Collector Settings
     # ---------------------------------------
     #
-    # These settings are enabled via the 'responsive-revision-gc' feature flag.
+    # These settings are enabled via the "responsive-revision-gc" feature flag.
     # ALPHA NOTE: This feature is still experimental and under active development.
+    #
+    # Active
+    #   * Revisions which are referenced by a Route are considered active and
+    #     are exempted from GC.
+    #   * Individual revisions may be marked with the annotation
+    #      "knative.dev/no-gc":"true" to be permanently considered active.
+    # Retention Time
+    #   * Revisions are deleted if they are:
+    #       1. not active
+    #       2. were created after more than "retain-since-create-time"
+    #       3. Were last referenced by a route more than
```

```suggestion
    #       3. were last referenced by a route more than
```
or the others to be capitalized.